### ISSUE TITLE
Create a dataset loader for RadGraph

### ISSUE THREAD

**Issue by hakunanatasha** [bigscience-workshop/biomedical#42]
From https://physionet.org/content/radgraph/1.0.0/

**Comment by alisoncallahan** [comment]
#self-assign

**Comment by alisoncallahan** [comment]
@hakunanatasha yes! still working on these - planning to dedicate most time to it next week (April 11-April 15). thanks!

**Comment by hakunanatasha** [comment]
@alisoncallahan wonderful thank you for letting me know! I'll assume the other datasets are also in-progress. Looking forward to your PRs!
